docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -309,6 +309,25 @@ Serialization Formats
         encryption. Choose this unless you have explicit legacy compatibility
         requirements.
 
+.. class:: PublicFormat
+
+    .. versionadded:: 0.8
+
+    An enumeration for public key formats. Used with the ``public_bytes``
+    method available on
+    :class:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPublicKeyWithSerialization`.
+
+    .. attribute:: SubjectPublicKeyInfo
```

Comment:
Ok. Good enough for me.

On Sun, Mar 8, 2015 at 3:12 PM, Paul Kehrer <notifications@github.com>
wrote:

> In docs/hazmat/primitives/asymmetric/serialization.rst
> <https://github.com/pyca/cryptography/pull/1706#discussion_r26007907>:
>
> > @@ -309,6 +309,25 @@ Serialization Formats
> >          encryption. Choose this unless you have explicit legacy compatibility
> >          requirements.
> >
> > +.. class:: PublicFormat
> > +
> > +    .. versionadded:: 0.8
> > +
> > +    An enumeration for public key formats. Used with the ``public_bytes``
> > +    method available on
> > +    :class:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPublicKeyWithSerialization`.
> > +
> > +    .. attribute:: SubjectPublicKeyInfo
>
> It's the name used by PyCrypto and Crypto++
> <http://www.cryptopp.com/wiki/Keys_and_Formats#Public_Key_Format>. Go
> calls it a "DER-encoded PKIX structure" but if you look in the source code
> it says "pkixPublicKey reflects a PKIX public key structure. See
> SubjectPublicKeyInfo in RFC 3280."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1706/files#r26007907>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084